qa/TL0_FW_iterators/test_mxnet.sh

```diff
@@ -21,8 +21,7 @@ test_body() {
         done
     fi
     ${python_invoke_test} -m '(?:^|[\b_\./-])[Tt]est.*mxnet*' test_fw_iterators_detection.py
-    ${python_invoke_test} -m '(?:^|[\b_\./-])[Tt]est.*mxnet*' test_fw_iterators.py
-    ${python_invoke_test} -m '(?:^|[\b_\./-])[Tt]est.*gluon*' test_fw_iterators.py
+    ${python_new_invoke_test} -A 'mxnet,gluon' test_fw_iterators
```

Comment:
I looked into the L0_TL0_FW_iterators--fw-mxnet job and it seems no tests were run for mxnet. Can you double check if it picks them correctly?

Reply:
There should be a separate job for that L0_TL0_FW_iterators--fw-mxnet

Reply:
Nice catch! It seems that comma-separated attributes don't work in nose2 (again, nose2 sucks big time).

Reply:
How about using expressions - https://docs.nose2.io/en/latest/plugins/attrib.html#select-tests-using-python-expressions ?

Reply:
According to https://github.com/nose-devs/nose2/blob/main/nose2/plugins/attrib.py if you pass a list of tuple of attributes it should work as well.

Reply:
I could, but is it really worth the complication? Using attributes (like in the updated code) works quite fine. My goal here was to simplify manual launches and especially filtering out unwanted frameworks. As I understand, it works like this:
```
-A 'a1,!a2' -A 'a3'
```
translates to `(a1 and not a2 ) or a3` - it makes perfect sense.